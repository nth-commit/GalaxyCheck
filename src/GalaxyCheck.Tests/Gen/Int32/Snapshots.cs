using Xunit;
using GC = GalaxyCheck;
using static Tests.TestUtils;

namespace Tests.Gen.Int32
{
    public class Snapshots
    {
        [Fact]
        public void Example() => SnapshotGenValues(GC.Gen.Int32());

        [Fact]
        public void ExampleOfExampleSpaces() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 10));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant1() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(0, 10).ShrinkTowards(5));

        [Fact]
        public void ExampleOfExampleSpaces_OriginVariant2() => SnapshotGenExampleSpaces(GC.Gen.Int32().Between(-10, 0).ShrinkTowards(-5));
    }
}
