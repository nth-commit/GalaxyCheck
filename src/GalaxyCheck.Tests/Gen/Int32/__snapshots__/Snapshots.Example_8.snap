[
  "1944859509",
  "626456108",
  "636234906",
  "-1962396394",
  "817040449",
  "-1122540840",
  "1521547617",
  "243997216",
  "1028091287",
  "1878519059",
  "-1840431918",
  "1473713115",
  "-810007966",
  "1749432806",
  "1236939830",
  "1222930174",
  "1357187263",
  "1148759760",
  "-1706178994",
  "-1470470884",
  "-560632095",
  "120859206",
  "1966810955",
  "-784201208",
  "-339925648",
  "297935958",
  "1535577277",
  "-1389608256",
  "906250308",
  "-299717978",
  "1087000963",
  "-1863658524",
  "-1338630383",
  "221807078",
  "-1422046698",
  "-1450972348",
  "-998658374",
  "1043572355",
  "-1478836560",
  "373166409",
  "-1536146725",
  "279441737",
  "1070414581",
  "86377478",
  "-712284895",
  "-1597601542",
  "-450511719",
  "55521046",
  "-320679194",
  "-774727380",
  "1264473609",
  "1534077254",
  "1417383966",
  "-27612066",
  "-2011878037",
  "582684505",
  "1817700967",
  "-76309340",
  "1302358657",
  "1758425690",
  "1772020656",
  "449043125",
  "-1956923830",
  "887649415",
  "1653226574",
  "-1280850262",
  "506754285",
  "-1731909033",
  "844852294",
  "-826841931",
  "577280111",
  "1316366905",
  "-389703997",
  "1621089346",
  "346549274",
  "1288779012",
  "-1101935888",
  "399590485",
  "-1979510887",
  "1638909748",
  "-590990152",
  "1122438290",
  "1202229355",
  "-1636258409",
  "210084867",
  "-1927421949",
  "2067486786",
  "-14171656",
  "1116203914",
  "1544595823",
  "-804588931",
  "1615264668",
  "-1991091115",
  "31049645",
  "1703501332",
  "-1804691582",
  "906723363",
  "-100010031",
  "-1178179462",
  "618316860"
]
